DOMAIN_CLASSIFICATION_GUIDE.md

```
@@ -0,0 +1,214 @@
+| **YOUTUBE** | YouTube 영상 | `/api/v1/summarize/youtube` |
+| **NAVER_NEWS** | 네이버 뉴스 | `/api/v1/summarize/naver-news` |
+| **TISTORY** | 티스토리 블로그 | `/api/v1/summarize/naver-news` |
+| **BRUNCH** | 브런치 | `/api/v1/summarize/naver-news` |
+| **GENERAL** | 기타 웹사이트 | `/api/v1/summarize/naver-news` |
+
+> **참고**: 네이버 뉴스, 티스토리, 브런치, 일반 웹사이트는 모두 웹 크롤링이 필요하므로 동일한 Python 엔드포인트를 사용합니다.
+
+## 🔍 URL 패턴 매칭
+
+### YouTube
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)